studies/models.py

```diff
@@ -186,6 +186,9 @@ class Study(models.Model):
         verbose_name=_('Hoe worden de deelnemers geworven?'))
     recruitment_details = models.TextField(
         _('Licht toe'),
+        help_text=_('Er zijn specifieke voorbeelddocumenten voor het gebruik van \
```

Comment:
This help text also appears a field too late, but here it's definitely an attribute to the wrong field

Reply:
Actually, I consciously chose to include it here, because this field appears only when you select Amazon Mechanical Turk (or 'other'), so it makes sense to have this specific information be made available when the user needs it?

Reply:
Ooh I didn't realize. Never mind, that makes sense.